parliament_challenge.py

```diff
@@ -0,0 +1,111 @@
+import requests
+import waitress
+import flask
+from flask import Flask, request, json
+
+
+def filter_speeches_dict(speeches):
+    """
+    Filter speeches data for relevant items.
```

Comment:
This is not correct. You're filtering speech data for the relevant "keys", not items.

And also say that the relevant keys are `anforande_id`, `dok_datum` .. and so on ..

Reply:
updated:

    Filtering speeches data for relevant keys.
    Relevant keys are anforande_id, dok_datum,
    parti, avsnittsrubrik, links and intressent_id.